examples/tutorials/advanced/non_ascii_text.py

```diff
@@ -0,0 +1,76 @@
+# ruff: noqa: RUF001,RUF003
+"""
+Typesetting non-ASCII text
+--------------------------
+
+In addition to ASCII printable characters, sometimes you may also want to typeset
+non-ASCII characters on the plot, such as Greek letters, mathematical symbols, or
+special characters.
+
+Due to the limitations of the underlying PostScript language, PyGMT doesn't support
+all characters in the Unicode standard. Instead, PyGMT supports a limited set of
+characters in the "Adobe Symbol", "Adobe ZapfDingbats", "Adobe ISOLatin1+", and
+"ISO-8859-*x*" (*x* can be 1-11, 13-16) encodings. Refer to :doc:`/techref/encodings`
+for the complete list of supported characters.
+
+In PyGMT, the supported (ASCII and non-ASCII) characters can be directly used in the
+``text`` parameter of the :meth:`pygmt.Figure.text` method for typesetting text strings.
+They can also be used in the arguments of other plotting functions (e.g., in the
+``frame`` parameter to set the labels or title).
+
+In this example, we demonstrate how to typeset non-ASCII characters in PyGMT.
+"""
+
+# %%
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[0, 5, 0, 6],
+    projection="X14c/7c",
+    frame=["xaf+lDistance (°)", "yaf+lValue (‰)", "WSen+tTitle: α² ± β²"],
+)
+
+fig.text(
+    x=[0.2, 0.2, 0.2, 0.2, 0.2],
+    y=[1, 2, 3, 4, 5],
+    text=["Mixed:", "ZapfDingbats:", "Symbol:", "ISOLatin1+:", "ASCII:"],
+    font="20p,Helvetica-Bold,red",
+    justify="LM",
+)
+fig.text(
+    x=[2, 2, 2, 2, 2],
+    y=[1, 2, 3, 4, 5],
+    text=[
+        "ABCD αβγδ ①②③ ➊➋➌",  # Mix characters from ISOLatin1+, Symbol and ZapfDingbats
+        "✈♥♦♣♠❛❜❝❞❨❩❪❫❬❭❮❯→↔",  # Non-ASCII characters from Adobe ZapfDingbats
+        "αβγδεζηθ⊗⊕∅⊃⊇⊄⊂⊆",  # Non-ASCII characters from Adobe Symbol
+        "±°ÀÁÂÃÄÅÈÌÒÙàèìòù",  # Non-ASCII characters from Adobe ISOLatin1+
+        "ABCDE12345!#$:;<=>?",  # ASCII only
+    ],
+    font="18p,Helvetica",
+    justify="LM",
+)
```

Comment:
I'm wondering if it is better to have the same vertical order for the text strings in the code and the text lines  in the generated plot:

| inverse vertical order |  same vertical order |
| --- | --- |
| ![text_nonascii_inversorder](https://github.com/user-attachments/assets/5460ae45-9277-40e7-b9d5-d2c15a133946) | ![text_nonascii_sameorder](https://github.com/user-attachments/assets/2ce49bf7-5e51-434e-8e2e-4d46009b6577) |


```suggestion
fig.text(
    x=[0.2, 0.2, 0.2, 0.2, 0.2],
    y=[5, 4, 3, 2, 1],
    text=["ASCII:", "ISOLatin1+:", "Symbol:", "ZapfDingbats:", "Mixed:"],
    font="20p,Helvetica-Bold,red",
    justify="LM",
)
fig.text(
    x=[2, 2, 2, 2, 2],
    y=[5, 4, 3, 2, 1],
    text=[
        "ABCDE12345!#$:;<=>?",  # ASCII only
        "±°ÀÁÂÃÄÅÈÌÒÙàèìòù",  # Non-ASCII characters from Adobe ISOLatin1+
        "αβγδεζηθ⊗⊕∅⊃⊇⊄⊂⊆",  # Non-ASCII characters from Adobe Symbol
        "✈♥♦♣♠❛❜❝❞❨❩❪❫❬❭❮❯→↔",  # Non-ASCII characters from Adobe ZapfDingbats
        "ABCD αβγδ ①②③ ➊➋➌",  # Mix characters from ISOLatin1+, Symbol and ZapfDingbats
    ],
    font="18p,Helvetica",
    justify="LM",
)

```